docs/reference/sections/dom-binding/eq.md

```diff
@@ -0,0 +1,9 @@
+`model.property | eq 'value'`
+
+Returns `true` if the property matches the passed in value.
+
+```html
+  <div rv-show="item.title | eq 'Shirt'">
+    //...
+  </div>
+```
```

Comment:
I think this structure works. Perhaps in the comment in the HTML example, you could add a description of what happens... so in this example, you could have:

```html
<div rv-show="item.title | eq 'Shirt'">
  <!-- Contents will be displayed when the item's title is 'Shirt' -->
</div>
```

Then for formatters that output values, you could have a comment indicating what the output would be, like:

```html
<span rv-text="cart.total_price | money"></span>
<!-- Displays as <span>$5.50</span> -->
```